openbts/core.py

```diff
@@ -151,14 +151,13 @@ def _send_and_receive(self, message):
         raw_response_data = self.socket.recv()
         return Response(raw_response_data)
       except zmq.Again:
-        # Reset the socket or it will be left in a bad state, waiting for a
-        # response.
-        self.socket.close()
-        self.setup_socket()
-        self.socket.connect(self.address)
-        raise TimeoutError('recv did not receive a response')
-    else:
-      raise TimeoutError('polling did not receive a response')
+        pass
+    # Reset the socket or it will be left in a bad state, waiting for a
+    # response.
```

Comment:
uber nit: can you add to this comment that we are now closing the socket if polling fails *or* if `recv` fails